lib/side-effect.js

```diff
@@ -24,7 +24,6 @@ export default function withSideEffect (reduceComponentsToState, handleStateChan
 
     const mountedInstances = new Set()
     let state
-    let shouldEmitChange = false
```

Comment:
This is a flag not to emit changes many times in a rendering cycle of React, but it's unneeded now.